app/react/Settings/components/EntityTypesList.js

```diff
@@ -72,14 +73,21 @@ export class EntityTypesList extends Component {
     );
   }
 
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
What about changing this method to something like `getSortedTemplates()` and instead of accepting a `templates` parameter, you just use `this.props.templates` right in the function body? That way we get rid of this eslint error and also make it more cohesive with the class.

Reply:
Done